src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+const allSpan = document.querySelectorAll('.population');
```

Comment:
it is always a good idea to choose a name for a variable from domain area - in this case it can be populationElements or something like this, feel free to choose more suitable name